_overviews/scala3-migration/scaladoc-settings-compatibility.md

```diff
@@ -0,0 +1,55 @@
+| -implicits-show-all | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-sound-shadowing | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-hide | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-debug | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-path | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams-max-classes | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams-max-implicits | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-timeout | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-restart | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -author | _ |  | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -raw-output | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -no-prefixes | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -skip-packages | -skip-packages |  |  | <i class="fa fa-check fa-lg"></i> |
+| -no-link-warnings | _ | | | <i class="fa fa-check fa-lg"></i> |
+| -expand-all-types | _ | | Setting should be removed from Scala2 scaladoc | <i class="fa fa-check fa-lg"></i> |
+| -groups | _ | | ~~Implement grouping as it is in Scala3. #11909~~ Actually we support grouping, e. g. [Future](https://scala3doc.virtuslab.com/pr-master/scala3/api/scala/concurrent/Future.html). Since one is using `@group` to actually group his methods, we should consider whether this flag is necessary. Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -no-java-comments | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -doc-canonical-base-url | _ |  A base URL to use as prefix and add `canonical` URLs to all pages. The canonical URL may be used by search engines to choose the URL that you want people to see in search results. If unset no canonical URLs are generated. | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -private | _ | Show all types and members. Unless specified, show only public and protected types and members. | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| _ | -siteroot | | We don't migrate it | <i class="fa fa-check fa-lg"></i> |
+| _ | -project-logo | | Should we migrate it? | <i class="fa fa-check fa-lg"></i> |
+| _ | -comment-syntax | | We don't migrate it | <i class="fa fa-check fa-lg"></i> |
```

Comment:
I guess the last column should be “N/A” (not applicable) for all the settings that didn’t exist in Scala 2?

Reply:
Good point, will change